public/app/plugins/panel/barchart/BarChartPanel.tsx

```diff
@@ -161,7 +163,40 @@ export const BarChartPanel = (props: PanelProps<Options>) => {
               getDataLinks={(seriesIdx, dataIdx) =>
                 vizSeries[0].fields[seriesIdx].getLinks?.({ valueRowIndex: dataIdx }) ?? []
               }
-              render={(u, dataIdxs, seriesIdx, isPinned, dismiss, timeRange2, viaSync, dataLinks) => {
+              getAdHocFilters={(seriesIdx, dataIdx) => {
```

Comment:
I get a suggestion in my IDE `'seriesIdx' is declared but its value is never read`

Should we remove this?